app/queries/content_change_matcher.rb

```diff
@@ -0,0 +1,7 @@
+class ContentChangeMatcher
+  def self.call(subscription:)
+    ContentChange.where(
```

Comment:
I think this works at the moment but I wonder if we'll need to update this to return something here that will expose which `SubscriberList` the `ContentChange`s relate to. Depends how we structure the email and implement the unsubscribe stuff.

Reply:
Yeah, I agree, we might need to return something more useful, but I wasn't sure what that would be yet.

Reply:
No, me neither... This works for a minimal implementation though so I'd go with it for now.